core/src/findbugs/findbugs-exclude.xml

```diff
@@ -15,4 +15,8 @@
     <Match>
         <Package name="~org\.jdbi\.v3\.core\.internal\.lexer\.*" />
     </Match>
+    <Match>
+        <!-- lifted from Google Guava -->
+        <Package name="~org\.jdbi\.v3\.core\.generic\.internal\.*" />
```

Comment:
I'd usually prefer to use the `@SuppressFBWarnings` annotation over excluding an entire package, but I'm guessing you did this so that we can have fewer local modifications?